src/eduid_webapp/oidc_proofing/tests/test_msgs.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+import unittest
+
+from eduid_webapp.oidc_proofing.helpers import OIDCMsg
+
+
+class MessagesTests(unittest.TestCase):
+    def test_messages(self):
+        """"""
+        self.assertEqual(str(OIDCMsg.temp_error.value), 'Temporary technical problems')
+        self.assertEqual(str(OIDCMsg.no_conn.value), 'No connection to authorization endpoint')
+        self.assertEqual(str(OIDCMsg.nin_invalid.value), 'nin needs to be formatted as 18|19|20yymmddxxxx')
```

Comment:
nin_invalid seems to be unused?

Reply:
You are right, I don't understand why, checking...

Also, I'm seeing I forgot to remove the useless `str()` from the tests :-/

Reply:
Ok now I remember, `'nin needs to be formatted as 18|19|20yymmddxxxx'` [comes from eduid-common](https://github.com/SUNET/eduid-common/blob/master/src/eduid_common/api/schemas/validators.py#L27). I wanted to gather all translatable messages in the enums, and this seemed better than having a loose enum in eduid-common.

If you agree with this, I'll add a comment to clarify.

Reply:
I agree in principle, but that would mean you would duplicate the common message in a number of these "return value enums"? That is IMO a step in the wrong direction.

Reply:
Not necessarily, since eduid-front does not distinguish the origin of the translatable messages it knows about, it is enough to have it just in one place.

Reply:
Well not just "not necessarily", but "not actually", I'm just keeping this message here =)

Reply:
Make an ENUM for common validation errors. We don't want non-OIDC microservices returning a message from OIDCMsg just because it happened to be defined there first.

Reply:
Ok, I agree it seems cleaner. I'll put it in eduid-common, alongside the base class `TranslatableMsg`.